src/ui/components/MapBox/MapBox.css

```diff
@@ -0,0 +1,33 @@
+.m-mapbox-body {
+  aspect-ratio: 16/9;
+  background-color: #f2f2f2;
+  position: relative;
+  width: 100%;
+}

```

Comment:
Isn't eslint showing you errors because there are no line breaks after each style closing brace?